### PR TITLE
Added info on how to find the keyID to the readme file.

### PR DESCRIPTION
**Pull request by genebean**
This module is very useful but I spent a fair amount of time figuring out how to find the keyID that it needs so I figured I'd add that to the readme file.

### CODE REVIEW

**Comment by stschulte** [comment]
@genebean thanks for letting me know it turns out to be useful in your environment. I didn't know about the gpg trick myself ;-)

File: README.md [hunk 1]
```diff
@@ -39,6 +39,13 @@ or - if you want to make sure a key is deleted - specify
     }
 
 The `name` of the `rpmkey` resource has to be the keyID of the gpg key.
+The keyID can be found via gpg by passing it the path to an existing key.
+For example, to find the keyID used by EPEL 7:
+
+    $ gpg ./RPM-GPG-KEY-EPEL-7
+    pub  4096R/352C64E5 2013-12-16 Fedora EPEL (7) <epel@fedoraproject.org>
+
+The sting after the / is what `rpmkey` expects (`352C64E5`).
```
Comment (stschulte) [inline README.md]: There is typo here (sting -> string). Depart from that it looks good. Can you amend your commit with the typo correction?

**Comment by genebean** [comment]
Fixed spelling... sorry 'bout that :)

**Comment by stschulte** [comment]
@genebean I've merged the README changes. Thank you very much for your contribution, it is much appreciated :-)

**Comment by genebean** [comment]
Awesome, glad to help!

**Pull request merged**